Assignment9/controller/books_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BooksController < ApplicationController
+  def index
+    @books = Book.all
+    render json: @books
+  end
+  def create
+    @books = Book.new(params.require(:book).permit(:title, :author, :genre, :quantity))
```

Comment:
Add new private method for book_params as:
`def book_params 
  params.require(:book).permit(:title, :author, :quantity)
end`

Reply:
done